wiki/osu!_wiki/en.md

```diff
@@ -0,0 +1,41 @@
+# osu! wiki
+
+The **osu! wiki** is an open-source knowledge database written and maintained by [volunteer contributors](https://github.com/ppy/osu-wiki/graphs/contributors). It contains a wide range of information on topics related to osu!, its community, and related activities and concepts. The wiki serves as an official source of information and is considered a documentation of the osu! history.
+
+All discussions which cover the wiki in general, its development, and changes to articles, happen in the `#osu-wiki` channel of the official [osu!dev Discord server](/wiki/osu!dev_Discord_server). To learn how you can help the osu! wiki, check the following articles:
+
+- [Contribution guide](/wiki/osu!_wiki_Contribution_Guide)
+- [Maintenance](/wiki/osu!_wiki/Maintenance)
+
+## History
+
+*See also: [History of the osu! wiki](/wiki/History_of_osu!/osu!_wiki)*
+
+The [first attempts](https://osu.ppy.sh/community/forums/posts/1175876) of creating a structured knowledge base are dated 2011, when [peppy](/wiki/People/peppy) has set up a MediaWiki instance for anyone willing to participate. Over the years, the maintenance and operation of MediaWiki has been proven difficult and toilsome, and in [August 26, 2016](https://discord.com/channels/188630481301012481/218677502141399041/218678017659109376) peppy and a team of volunteers have begun the process of migrating the osu! wiki to GitHub, where it resides to this day. The wiki has then been integrated into the new osu! website and received technical and visual updates critical for good user experience.
```

Comment:
```suggestion
The [first attempts](https://osu.ppy.sh/community/forums/posts/1175876) of creating a structured knowledge base are dated to 2011, when [peppy](/wiki/People/peppy) set up a [MediaWiki](https://en.wikipedia.org/wiki/MediaWiki) instance for anyone willing to participate. Over the years, maintenance and operation of the MediaWiki has been proven difficult and toilsome, and in [August 26, 2016](https://discord.com/channels/188630481301012481/218677502141399041/218678017659109376) peppy and a team of volunteers began the process of migrating the osu! wiki to GitHub, where it resides to this day. The wiki has since been integrated into the new osu! website and received technical and visual updates critical for good user experience.
```

Reply:
sigh. thanks!